.github/workflows/submitty_ci.yml

```diff
@@ -468,11 +474,9 @@ jobs:
         with:
           config: baseUrl=http://localhost
           working-directory: ${{env.SUBMITTY_REPOSITORY}}/site
-          spec: cypress/e2e/cypress-feature/*
+          spec: cypress/e2e/Cypress-${{matrix.containers}}/*
           browser: chrome
           env: run_area=CI
-          record: true
-          parallel: true  # Runs test in parallel using settings above
```

Comment:
(Leaving this here since I can't comment on lines you didn't edit)

Are the env variables under this necessary now?

Reply:
No they are not, I will remove them tonight.